extensions/amp-render/1.0/amp-render.js

```diff
@@ -71,6 +71,24 @@ const getAmpStateJson = (element, src) => {
     });
 };
 
+/**
+ * @param {string} bindingValue
+ * @param {boolean} isFirstMutation
+ * @return {boolean} Whether bind should evaluate and apply changes.
+ */
+const getUpdateValue = (bindingValue, isFirstMutation) => {
```

Comment:
Let's make this a named function.

```suggestion
function getUpdateValue(bindingValue, isFirstMutation) {
```